test/StatementsTest.chpl

```diff
@@ -0,0 +1,53 @@
+use UnitTest;
```

Comment:
Add a test when setValue is called with wrong placeholder index.

Reply:
Also add a test when some larger index is set before a small one.

Reply:
Sure, I'll add a test for the first scenario, but it should be kept in mind that there's no way to test an exception currently. Hence, I'll be modifying things to display a warning in case setValue is called on an invalid index (halting seems a little extreme in this case in my opinion), do let me know if that looks good.
I'll add a test for the second scenario you mentioned.

Reply:
Let's have a halt and todo for test.

Reply:
Make sure you just check the halt locally.

Reply:
Sure, I'll add a halt and test it locally.